docs/getting-started.md

```diff
@@ -31,9 +31,21 @@ If these don't work, more info is in [#805](https://github.com/StackExchange/dns
 
 Get prebuilt binaries from [GitHub releases](https://github.com/StackExchange/dnscontrol/releases/latest)
 
-Alternatively, on Mac you can install it using homebrew:
+## Through a package manager
```

Comment:
Suggestion: Give the package instructions first, then the source code instructions.  That way we encourage the use of the packages.